content/components/relative-time.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Relative time
+description: Relative time formats dates and times into a human-readable string.
+reactId: relative_time
+componentId: relative_time
+---
+
+## Usage
+
+This component does not include any styling. It formats a timestamp as a localized string or as relative text that auto-updates in the user's browser.
+
+For more information about Primer's patterns for date and tme formatting, see the "[Dates and times](/ui-patterns/dates-and-times)" UI pattern guidelines.
+
+## Options
+
+### Verbosity
+
+**Auto**: "Apr 1, 2014"
+
+**Micro**: "9y"
```

Comment:
The micro pattern is not recommended due to issues with screen readers mispronouncing "9m" as "9 meters". We should discourage its use.

Reply:
Agree with this, we cover that in the Date & Times docs: https://primer.style/design/ui-patterns/dates-and-times#responsive-design-and-micro-format

Reply:
@josepmartins - do you want to create an issue to remove the `"micro"` value as an option for the `format` prop?